### DISCUSSION TITLE
5.0.0版本兼容安卓 5版本吗

### DISCUSSION THREAD

**Discussion by 123liyuan123** [ant-design/ant-design-mobile#5523] [General]
5.0.0版本兼容安卓 5版本吗

**Comment by awmleer** [comment]
安卓系统的兼容性取决于使用的具体浏览器版本，关于兼容性，详细的介绍在[文档](https://mobile.ant.design/zh/guide/quick-start#%E5%85%BC%E5%AE%B9%E6%80%A7)上
